lightning/src/ln/channel.rs

```diff
@@ -138,7 +138,7 @@ enum FeeUpdateState {
 enum InboundHTLCRemovalReason {
 	FailRelay(msgs::OnionErrorPacket),
 	FailMalformed(([u8; 32], u16)),
-	Fulfill(PaymentPreimage),
+	Fulfill(PaymentPreimage, Option<AttributionData>),
```

Comment:
To clarify -- we need to include the `hmacs` part in the fulfills to verify the hold times part?